snapcraft.yaml

```diff
@@ -151,6 +151,7 @@ parts:
       bin/subiquity-service: usr/bin/subiquity-service
       bin/subiquity-server: usr/bin/subiquity-server
       bin/subiquity-cmd: usr/bin/subiquity-cmd
+      $SNAPCRAFT_PART_BUILD/system_setup/ubuntu-wsl-setup: bin/ubuntu-wsl-setup
```

Comment:
Maybe we should make it not runnable outside of WSL (exiting non-zero if WSL is not detected).

Reply:
I made it just in case.

Reply:
This is fine.
Not as a blocker for merge, I will be quite curious to see this in action.  What would I need to test this out?

Reply:
You need a Windows machine with WSL 2 and an instance of Ubuntu, preferably Kinetic. In a couple of days, I expect those changes mentioned in this PR to be seeded in our rootfs. Testing this will only require running:
`sudo /usr/libexec/wsl-setup --text`. That mounts the snap and runs `system_setup.cmd.tui`.
To test outside of WSL (on a LXD container, for instance) you need to remove the recently added checking for WSL2 (or fool it by creating the WSLInterop fie :D). Then you need to install subiquity snap and the wsl-setup deb package. As before running `sudo /usr/libexec/wsl-setup --text` mounts the snap and runs `system_setup.cmd.tui`.